### PR TITLE
Kernel: Fix incorrect indentation

### PR DESCRIPTION
**Pull request by gunnarbeutner**
Looks like a tab snuck in. :)

cc @alimpfard :stuck_out_tongue:

### CODE REVIEW

File: Kernel/Arch/x86/common/Interrupts.cpp [hunk 1]
```diff
@@ -68,7 +68,7 @@ static EntropySource s_entropy_source_interrupts { EntropySource::Static::Interr
     }
 
 #define EH_ENTRY_NO_CODE(ec, title)                                            \
-    extern "C" void title##_asm_entry();	                                   \
+    extern "C" void title##_asm_entry();                                       \
     extern "C" void title##_handler(TrapFrame*) __attribute__((used));         \
     NAKED void title##_asm_entry(){                                            \
```
Comment (alimpfard) [inline Kernel/Arch/x86/common/Interrupts.cpp:73]: A lot more snuck in, such as this brace not having a space before it.
Maybe take this out of the macro, reformat it (sans the asm), then put it back?

Comment (alimpfard) [reply Kernel/Arch/x86/common/Interrupts.cpp:73]: if you don't feel like it, I could do that myself now :)

Comment (gunnarbeutner) [reply Kernel/Arch/x86/common/Interrupts.cpp:73]: Oof, hadn't even noticed that. Hang on.

**Comment by Hendiadyoin1** [comment]
I was wondering why the github preview was different from what vs code showed me....

This was probably my fault...

**Comment by gunnarbeutner** [comment]
Eh, the formatting was worse before. :)

**Review by alimpfard — APPROVED** [review]
Looks good!

**Pull request merged**